src/5.0_to_5.1/calculated-measures/migrateSavedCalculatedMeasures.test.ts

```diff
@@ -1,25 +1,16 @@
-import { migrateCalculatedMeasures } from "./migrateCalculatedMeasures";
-import { sandboxDataModel } from "@activeviam/data-model-5.1/dist/__test_resources__";
+import { migrateSavedCalculatedMeasures } from "./migrateSavedCalculatedMeasures";
 import { contentServer } from "../__test_resources__/contentServer";
 import { uiCalculatedMeasuresFolder } from "../__test_resources__/uiCalculatedMeasuresFolder";
-import { uiDashboardsFolder } from "../__test_resources__/uiDashboardsFolder";
-import { uiWidgetsFolder } from "../__test_resources__/uiWidgetsFolder";
 import { ErrorReport, OutcomeCounters } from "../../migration.types";
 import _cloneDeep from "lodash/cloneDeep";
 import _fromPairs from "lodash/fromPairs";
 
-const dataModels = {
-  "Ranch 6.0": sandboxDataModel,
-  "Ranch 5.11": sandboxDataModel,
-};
 const contentServerForTests = _cloneDeep(contentServer);
 const errorReport: ErrorReport = {};
 
 contentServerForTests.children!.ui.children = {
   ...contentServerForTests.children!.ui.children,
   calculated_measures: uiCalculatedMeasuresFolder,
```

Comment:
Ultimately we could, and same for the `auiDashboardsFolder` that was there before (instead of removing it).
But this is even more refactor, and I prefer to keep this for later.

Reply:
Sounds fair!